layouts/partials/implementations.html

```diff
@@ -44,12 +43,24 @@ <h5 class="content-center">Desktop App</h5>
           <div class="app-ss content-center">
             <img src="/images/ipfs-companion-ss.png" />
           </div>
-          <h5 class="content-center">Browser Companion</h5>
-          <p align="left">Upgrade your browser with IPFS super powers. This add-on enables everyone to access IPFS resources the way they
-            were meant to.</p>
+          <h5 class="content-center">Browser Extension</h5>
+          <p align="left">
+            Upgrade your browser with IPFS superpowers. The IPFS Companion Web Extension connects your local IPFS daemon with your browser, enabling ipfs:// scheme urls and more!
```

Comment:
I changed `super powers` to `superpowers`. I believe that's the internet-accepted way to go :)